src/Types/RecoilAtomTypes.res

```diff
@@ -2,4 +2,5 @@ type field = {
   value: string,
   isValid: option<bool>,
   errorString: string,
+  countryCode?: string,
```

Comment:
Why are we adding country code in the generic field type? Is it not possible to use value?

Reply:
Because dropdown is different component and text field where number will be inputed that's a different component. I need some different way to add this and send. I'll try to change but most probably it will be like this I guess.